packages/edit-post/src/components/visual-editor/index.js

```diff
@@ -70,7 +70,6 @@ function MaybeIframe( {
 
 	return (
 		<Iframe
-			headHTML={ window.__editorStyles.html }
 			head={ <EditorStyles styles={ styles } /> }
```

Comment:
Will Iframe's inputs be consolidated in the future? With this change, we now have:

- `head` prop to pass `settings.styles`
- a hard-coded `<style>{ 'body{margin:0}' }</style>` injected with the `head` prop
- `__editorAssets` global to pass other styles and scripts

It shouldn't block this PR, but it seems like something to address at some point.

Reply:
After this PR, we have one less? :) The hardcoded style was there before, it's the same a mini default stylesheet. `settings.styles` can be built-in in the future.